specs/AppLifecycle/Activation/AppLifecycle Activation.md

```diff
@@ -245,8 +252,9 @@ unregister for these activation kinds.
 void UnregisterForActivation()
 {
     // Unregister one or more registered filetypes.
+    string imageFileTypes[3] = { ".jpg", ".png", ".bmp" };
     ActivationRegistrationManager::UnregisterForFileTypeActivation(
-        new string[]{ ".jpg", ".png", ".bmp" });
+        imageFileTypes);
 
     ActivationRegistrationManager::UnregisterForProtocolActivation("foo");
     ActivationRegistrationManager::UnregisterForStartupActivation();
```

Comment:
Don't we need to specify which taskId we're unregistering?

Reply:
Fixed.